app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_name(command: str) -> None:
```

Comment:
The function name should be `copy_file` instead of `copy_name` to match the task description.